packages/datadog-instrumentations/src/openai.js

```diff
@@ -278,7 +287,18 @@ for (const shim of V4_PACKAGE_SHIMS) {
         // The OpenAI library lets you set `stream: true` on the options arg to any method
         // However, we only want to handle streamed responses in specific cases
         // chat.completions and completions
-        const stream = streamedResponse && arguments[arguments.length - 1]?.stream
+        const stream = streamedResponse && getOption(arguments, 'stream', false)
+
+        // we need to compute how many prompts we are sending in streamed cases for completions
+        // not applicable for chat completiond
+        let n
+        if (stream) {
+          n = getOption(arguments, 'n', 1)
+          const prompt = getOption(arguments, 'prompt')
+          if (Array.isArray(prompt) && typeof prompt[0] !== 'number') {
+            n *= prompt.length
```

Comment:
Just confirming for my own understanding, is this similar logic to what's happening on the Python side? https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/contrib/openai/_endpoint_hooks.py#L123-L128

Reply:
yep!